source/common/config/well_known_names.cc

```diff
@@ -210,6 +210,9 @@ TagNameValues::TagNameValues() {
 
   // http.<stat_prefix>.rbac.(<rules_stat_prefix>.)*
   addTokenized(RBAC_HTTP_PREFIX, "http.*.rbac.$.**");
+
+  // listener.[<address>.]downstream_cx_proxy_proto.versions.(<version>).**
+  addRe2(PROXY_PROTOCOL_VERSION, R"(downstream_cx_proxy_proto\.versions(\.(<TAG_VALUE>)))", "");
```

Comment:
`addTokenized` is much faster/more-efficient. I think you can use that in this case and get the correct value out.

Reply:
Done